posthog/tasks/exports/csv_exporter.py

```diff
@@ -179,7 +183,19 @@ def _export_to_csv(exported_asset: ExportedAsset, limit: int = 1000, max_limit:
             raise Exception(f"export API call failed with status_code: {response.status_code}. {response_json}")
 
         # Figure out how to handle funnel polling....
-        data = response.json()
+        data = []  # response.json()
```

Comment:
Should this line be reverted?

Reply:
Ha, obviously not 🤣 

Damn, remembered to revert when I hard coded None but then relaxed before realising I should check that `[]` was ok.